extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -596,7 +598,8 @@ const viewContent = () => {
   margin-top: 1.33em;
 }
 
-.deployment-summary {
+.deployment-summary-title {
+  margin-block-start: 1.33em;
```

Comment:
This was pulled out of the user stylesheet of VS Code to unify it with `.progress-ring`